implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v2/DynatraceExporterV2Test.java

```diff
@@ -346,6 +343,110 @@ void toLongTaskTimerLine() {
+        // this test we need to use the system clock.
+        Clock clock = Clock.SYSTEM;
+        DynatraceConfig config = createDefaultDynatraceConfig();
+        DynatraceMeterRegistry registry = DynatraceMeterRegistry.builder(config).clock(clock).build();
+        DynatraceExporterV2 exporter = new DynatraceExporterV2(config, clock, httpClient);
+
+        LongTaskTimer ltt = LongTaskTimer.builder("my.ltt").register(registry);
+
+        // This will run in the background until it is interrupted down below.
+        ExecutorService executorService = Executors.newSingleThreadExecutor();
+        executorService.submit(() -> busyWait(ltt));
+
+        // wait for a little bit to ensure there is something to measure
+        Thread.sleep(500);
+
+        List<String> lines = exporter.toLongTaskTimerLine(ltt).collect(Collectors.toList());
+
+        // export complete, can shut down active background thread.
+        executorService.shutdownNow();
+
+        // assertions
+        assertThat(lines).hasSize(1);
+
+        // A String[]: "min=1", "max=1", "sum=1", "count=1"
+        String[] gaugeComponents = lines.get(0).split(" ")[1].split(",");
+        // trim the "min=", "max=", etc. prefixes and keep only the string representations
+        // of the values.
+        String min = gaugeComponents[1].split("=")[1];
+        String max = gaugeComponents[2].split("=")[1];
+        String sum = gaugeComponents[3].split("=")[1];
+        String count = gaugeComponents[4].split("=")[1];
+
+        assertThat(min).isEqualTo(sum).isEqualTo(max);
+
+        assertThat(count).isEqualTo("1");
+    }
+
+    @Test
+    void longTaskTimerWithMultipleValuesExportsConsistentData() throws InterruptedException {
+        // For this test we need to use the system clock. See
+        // longTaskTimerWithSingleValueExportsConsistentData for
+        // more info
+        Clock clock = Clock.SYSTEM;
+        DynatraceConfig config = createDefaultDynatraceConfig();
+        DynatraceMeterRegistry registry = DynatraceMeterRegistry.builder(config).clock(clock).build();
+        DynatraceExporterV2 exporter = new DynatraceExporterV2(config, clock, httpClient);
+
+        LongTaskTimer ltt = LongTaskTimer.builder("my.ltt").register(registry);
+
+        // This will run in the background until it is interrupted down below.
+        ExecutorService executorService = Executors.newFixedThreadPool(2);
+        executorService.submit(() -> busyWait(ltt));
+        Thread.sleep(500);
+        executorService.submit(() -> busyWait(ltt));
+
+        Thread.sleep(500);
+
+        List<String> lines = exporter.toLongTaskTimerLine(ltt).collect(Collectors.toList());
+
+        // export complete, can shut down active background thread.
+        executorService.shutdownNow();
+
+        // assertions
+        assertThat(lines).hasSize(1);
+
+        // A String[]: "min=1", "max=1", "sum=1", "count=1"
```

Comment:
```suggestion
        // A String[]: "min=x", "max=x", "sum=x", "count=x"
```
We don't know the values here right? So maybe leave them "unset" as saying "1" might be misleading?